api/networking/v1alpha1/virtualip_types.go

```diff
@@ -55,8 +65,15 @@ type VirtualIPList struct {
 	Items           []VirtualIP `json:"items"`
 }
 
+type EphemeralVirtualIPSpec struct {
+	// VirtualIPSpec defines the desired state of VirtualIP
+	VirtualIPSpec `json:",inline"`
+	// ReclaimPolicy is the ReclaimPolicyType of virtualIP
+	ReclaimPolicy ReclaimPolicyType `json:"reclaimPolicy,omitempty"`
```

Comment:
What should happen if I don't provider a Reclaim policy? Do we default to `Delete`? Or do we implicitly use the `Delete` behaviour?

Reply:
We don't set the default `ReclaimPolicy` explicitly to `Delete`, It's considered implicit `Delete` to avoid breaking current behavior.